nengo_gui/static/hotkeys.js

```diff
@@ -1,3 +1,11 @@
+//Initializes all hotkey behaviour
```

Comment:
Hmm, this extra comment here at the top seems a bit odd to me.  Is it meant to indicate something different from the comment 3 lines below it?